rewrite-java/src/main/java/org/openrewrite/java/JavadocPrinter.java

```diff
@@ -376,10 +384,33 @@ public J visitTypeParameter(J.TypeParameter typeParam, PrintOutputCapture<P> p)
 
         @Override
         public Space visitSpace(Space space, Space.Location loc, PrintOutputCapture<P> p) {
-            p.append(space.getWhitespace());
+            List<Javadoc.LineBreak> lineBreaks = getCursor().getNearestMessage("LINE_BREAKS");
```

Comment:
`LINE_BREAKS` probably should be qualified to prevent name collision, I know it's not likely...but it could happen. 

`JAVADOC_LINE_BREAKS`

And `INDEX` to `JAVADOC_LINE_BREAK_INDEX`

Reply:
That's fair, better to be safe